platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/config/ReconnectConfig.java

```diff
@@ -60,7 +60,7 @@ public record ReconnectConfig(
         @ConfigProperty(defaultValue = "-1") int reconnectWindowSeconds,
         @ConfigProperty(defaultValue = "0.50") double fallenBehindThreshold,
         @ConfigProperty(defaultValue = "300s") Duration asyncStreamTimeout,
-        @ConfigProperty(defaultValue = "100ms") Duration asyncOutputStreamFlush,
+        @ConfigProperty(defaultValue = "8ms") Duration asyncOutputStreamFlush,
```

Comment:
just curios - why 8ms? where did this number come from?

Reply:
I can't say for sure where the original value `100ms` came from. In one of my runs I found that flushing every 100ms is too slow, and reduced it to 8ms. Not that I needed 8ms in particular, just some smaller value than 100ms. This has greater effect when network latency is low, and lesser effect, if latency is high